app/models/episode_story_handler.rb

```diff
@@ -0,0 +1,95 @@
+require 'episode'
+
+class EpisodeStoryHandler
+  include PRXAccess
+
+  attr_accessor :episode, :story
+
+  def initialize(episode)
+    self.episode = episode
+  end
+
+  def self.create_from_story!(story)
+    series_uri = story.links['series'].href
+    podcast = Podcast.find_by!(prx_uri: series_uri)
+    episode = Episode.new(podcast: podcast)
+    update_from_story!(episode, story)
+  end
+
+  def self.update_from_story!(episode, story = nil)
+    story ||= get_story
+    new(episode).update_from_story!(story)
+  end
+
+  def update_from_story!(story)
+    Episode.transaction do
+      episode.lock!
+      update_from_story(story)
+      episode.save!
+    end
+    episode
+  end
+
+  def update_from_story(story)
+    self.story = story
+    self.episode.prx_uri = story.links['self'].href
```

Comment:
do you have a better suggestion what we should store to correlate with the CMS stories?
Didn't seem like it would be very hypermedia to use the story id, but we do have that